plugins/mistral/manifest.json

```diff
@@ -0,0 +1,66 @@
+{
+  "id": "mistral",
+  "description": "Mistral Content Moderation classifier leverages the most relevant policy categories for effective guardrails and introduces a pragmatic approach to LLM safety by addressing model-generated harms such as unqualified advice and PII",
+  "credentials": {
+    "type": "object",
+    "properties": {
+      "apiKey": {
+        "type": "string",
+        "label": "API Key",
+        "description": "Find your API key in the Mistral la-plateforme",
+        "encrypted": true
+      }
+    },
+    "required": ["apiKey"]
+  },
+  "functions": [
+    {
+      "name": "Moderate Content",
+      "id": "moderateContent",
+      "type": "guardrail",
+      "supportedHooks": ["beforeRequestHook"],
```

Comment:
Is there a reason to only support beforeRequestHook?

Reply:
there's no specific reason, just replicated the same schema we have for the `portkey` plugin.

Reply:
That could be a bug for portkey plugin manifest. We can add it here because the handler is anyways supporting it.